README.md

```diff
@@ -27,9 +27,54 @@ described in the [integration guide](http://developer.gini.net/gini-sdk-ios/docs
 is provided at the instance's `injector` property) to create the manager instances and to manage the dependencies
 between the managers and makes those manager instances available as properties.
 
-## Requirements
+## Public Key pinning
+The Gini iOS SDK allows you to enable public key pinning with the Gini API through the [TrustKit](https://github.com/datatheorem/TrustKit/) library. In case you want to implement it, you just need to pass a dictionary with all the parameters required into one of the `GINISDKBuilder` initializers, as follows:
+
+**Swift**
+```swift
+let trustKitConfig = [
+            kTSKSwizzleNetworkDelegates: false,
+            kTSKPinnedDomains: [
+                "example.com": [
+                    kTSKExpirationDate: "2017-12-01",
```

Comment:
In the Android TrustKit implementation, if public key pinning is not enforced, TrustKit will do a report about public key mismatch, but it won't fail the server check, effectively disabling pinning.

Can you check in the iOS implementation for that and if the behavior is the same enable `kTSKEnforcePinning` here, too?

Reply:
Yes, there is that field also on iOS, I was using it to test it but I forgot to add it to the documentation. 💪